glossary.md

```diff
@@ -18,6 +22,10 @@ BDSM is short for Bondage, Discipline, Dominance, Submission, Sadism, and Masoch
 
 A narrative path within a visual novel that focuses on developing the relationship between the player character and a specific in-game character.
 
+## Canon Scene
```

Comment:
Let's change this to canon event and add a description like "Derived from the term [canon](https://en.wikipedia.org/wiki/Canon_(fiction))".